README.md

```diff
@@ -1 +1,76 @@
 # Physlr: Construct a Physical Map from Linked Reads
+
+Physlr is a *de novo* physical map generater using linked reads (10X or stLFR). This physical
+map can then be used to scaffold an existing assembly to yield chromosomal level contiguity.
+
+## Build Status
+[![Build Status](https://dev.azure.com/bcgsc/btl/_apis/build/status/bcgsc.physlr?branchName=master)](https://dev.azure.com/bcgsc/btl/_build/latest?definitionId=1&branchName=master)
```

Comment:
I'm not personally a fan of build status badges in a README, just my 2c.

Reply:
I do not have a strong opinion either way but I just see a lot of repo with it so I added it to the README. Why are you not a fan of the badge?

Reply:
A build status badge is mean to be developer-facing not user-facing, and `README.md` is a user-facing document.

Reply:
Hmm, I see. I will remove it then, but wouldn't the badge also give the user a source of confidence if he were to clone from master?